README.md

```diff
@@ -49,7 +49,8 @@ final url = Uri.https('accounts.google.com', '/o/oauth2/v2/auth', {
   'redirect_uri': '$callbackUrlScheme:/',
   'scope': 'email',
 });
-
+// google multiple scopes 
+'scope':'https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/calendar.events '
```

Comment:
This makes the dart code invalid so I don't think that this is the best place to put it 🤔